apps/web/public/static/locales/en/common.json

```
@@ -2561,5 +2561,8 @@
   "number_of_options":"{{count}} options",
   "reschedule_with_same_round_robin_host_title": "Reschedule with same Round-Robin host",
   "reschedule_with_same_round_robin_host_description": "Rescheduled events will be assigned to the same host as initially scheduled",
+  "choose_hosts_schedule" : "Choose hosts schedule",
```

```suggestion
  "choose_hosts_schedule" : "Choose host schedules",
```
I think that's better

---

updated